compiler/modules/comm.diagnostic/printer.bal

```diff
@@ -0,0 +1,88 @@
+import ballerina/io;
+import ballerina/file;
+
+public type Error error<Diagnostic>;
+
+public type Printer object {
+    public function print(Error diagnostic);
```

Comment:
The argument to print should be a Diagnostic not an Error.

Reply:
fixed